bean/internal/usecase/membership/membership.go

```diff
@@ -67,3 +67,21 @@ func (u *UseCase) Validate(userID string) (bool, error) {
 
 	return true, nil
 }
+
+func (u *UseCase) findOrCreateUser(email string) (*model.User, error) {
```

Comment:
This is a duplicate from the passwordless use case.

It's just two usages. It's easier to repeat ourselves right now.